envoy/datadog_checks/envoy/metrics.py

```diff
@@ -3140,6 +3182,38 @@
         ),
         'method': 'monotonic_count',
     },
+    'cluster.http1.dropped_headers_with_underscores': {
+        'tags': (
+            ('cluster_name', ),
+            (),
+            (),
```

Comment:
Why two extra `(),`?

Reply:
Each metric part has a tuple of 0 or more tags